go/loopback/main.go

```diff
@@ -32,49 +37,72 @@ func Init(homeDir string, logFile string, runModeStr string, serverURI string, a
 			fmt.Println("Error decoding run mode", err, runModeStr)
 		}
 		config := libkb.AppConfig{HomeDir: homeDir, LogFile: logFile, RunMode: runMode, Debug: true, LocalRPCDebug: "Acsvip", ServerURI: serverURI, SecurityAccessGroupOverride: accessGroupOverride}
-		err = libkb.G.Configure(config, usage)
+		err = kbCtx.Configure(config, usage)
 		if err != nil {
 			panic(err)
 		}
 
-		service := (service.NewService(g, false))
-		service.StartLoopbackServer()
-		service.G().SetService()
+		svc := service.NewService(kbCtx, false)
+		svc.StartLoopbackServer()
+		kbCtx.SetService()
+		kbCtx.SetUIRouter(service.NewUIRouter(kbCtx))
 
 		serviceLog := config.GetLogFile()
 		logs := libkb.Logs{
 			Service: serviceLog,
 		}
 
 		logSendContext = libkb.LogSendContext{
-			Contextified: libkb.NewContextified(service.G()),
+			Contextified: libkb.NewContextified(kbCtx),
 			Logs:         logs,
 		}
 
+		// Uncomment after we KBFS is ready
+		// kbfsParams := libkbfs.DefaultInitParams(kbCtx)
```

Comment:
Oh also, that code that's commented out does work and had iPhone doing fs listing :)  ... I'm just temporarily commenting out so that Circle CI passes... (I need to commit this and KBFS first and then uncomment that in separate PR).